### DISCUSSION TITLE
JSONP Request as POST

### DISCUSSION THREAD

**Discussion by mrprogramator** [jquery/jquery#5472] [General]
Hey there! I need to make a jsonp request to send cross-domain recorded audio, as a blob, to other service, but jsonp only allows get request. As you can imagine, audio becomes very large, and get-requests have a limited request length. If you have any idea of how to solve this, it would be very helpful.

**Comment by dmethvin** [comment]
JSONP is basically a `<script>` tag so the only option is GET. The server would need to set a CORS header to allow other request methods.
